setup.py

```diff
@@ -12,6 +12,7 @@
 install_requires = [
     'wagtail>=1.0',
     'django-model-utils>=2.5.2',
+    'django-request-provider==1.0.2',
```

Comment:
Uit de docs: "You shouldn't be using this project. But you should be using the real one which is" https://pypi.python.org/pypi/django-contrib-requestprovider/1.0.1